packages/apps/tools/src/components/Common/Layout/partials/Sidebar/Toolbar.tsx

```diff
@@ -21,19 +22,27 @@ export const Toolbar: FC = () => {
     isMenuOpen,
     visibleLinks,
     setVisibleLinks,
+    setIsMenuOpen,
   } = useLayoutContext();
   const { pathname } = useRouter();
 
-  const handleItemClick = (index: number): void => {
+  const handleItemClick = (
+    event: React.MouseEvent<HTMLButtonElement, MouseEvent>,
+    index: number,
+  ): void => {
+    event.preventDefault();
```

Comment:
Out of curiosity; why do we need to do this?

Reply:
to prevent refresh of the page

Reply:
Ah, right 💡